### ISSUE TITLE
2020-01-07 tc120 M3SS missing DOSE2U definition causes crash

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#149]
tc120 M3SS is based upon  B2901001 with 2 defined dosing intervals. However, there's only one DOF/DOFU defined (need to confirm that the DOF defined is used for both intervals and whether a second definition DOF2/DOF2U would be used if indeed incoporated into the MCT, note that output for DOF2 appears in the parameter results) and only one DOSE1/DOSE1U pair defined.

In this case, DOSE2 is assumed to be unit dose, which is expected. However, the run_M3_SS module crashes since DOSE2U is not defined. If DOSE2U is defined, the crash doesn't occur. 

Recommend that if DOSE2 is not defined and assume that it will be unit dose, the associated DOSE2U lack of definition not cause a crash. Permit this to be missing.

tc120 files will be added to onedrive for testing.

**Comment by jhhughes256** [comment]
Testing with commit ed0aed3 an error still occurs.

```r
> paste(testcase.id, mct$MODEL, mct$DOSINGTYPE, mct$AUCMETHOD, mct$TIME, sep = ".")
[1] "tc120.M3.SS.LINLOG.NOMINAL"
> # pretime <- Sys.time()
> results_list <- run_computation(data = d, map = mct, flag = flags, 
+   parameterset = parameterset)
DOSE2  as defined in 'map', do not appear in input concentration dataset 
assuming unit dose amounts for:  DOSE2 
DOSE2U  as defined in 'map', do not appear in input concentration dataset 
Error in value[[3L]](cond) : 
  Error in `[.data.frame`(data_data, !duplicated(data_data[, xdose]), vlist): undefined columns selected
For SDEID 49420991
In addition: Warning messages:
1: In validate_timeconc_data(map, data, flag, verbose = verbose) :
  assuming unit dose amounts for each dose
2: In validate_timeconc_data(map, data, flag, verbose = verbose) :
  Dosing unit: 'DOSE2U' isn't present in input concentration dataset

3: In doTryCatch(return(expr), name, parentenv, handler) :
  Removing SDEID: '49420991' due to duplicate TIME but different CONC values
4: In doTryCatch(return(expr), name, parentenv, handler) :
  Missing concentration at TAU and/or TOLD for SDEID: '49420991'
5: In doTryCatch(return(expr), name, parentenv, handler) :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in `[.data.frame`(data_data, !duplicated(data_data[, xdose]), vlist): undefined columns selected
For SDEID 49420991 
```

**Comment by jhhughes256** [comment]
This issue has been closed from a scope perspective, but not from a GitHub perspective.

**Comment by jhhughes256** [comment]
While not officially in scope, this issue has been addressed due to other fixes with commit 53d87f1.
